kbase-extension/scss/modules/_typography.scss

```diff
@@ -1,35 +1,45 @@
 @mixin font-h1 {
-  font: normal 700 42px $typeface-display-legacy;
+  font: normal 700 42px $typeface-display;
```

Comment:
Migrated these over to using the new display typeface so we can start using these mixins!